nucleus/payara-modules/hazelcast-bootstrap/src/main/java/fish/payara/nucleus/hazelcast/MemberAddressPicker.java

```diff
@@ -155,6 +155,14 @@ static InetSocketAddress initAddress(String address, int port) {
         }
     }
 
+    static InetAddress findMyAddressOrLocalHost() {
+        InetAddress myAddress = findMyAddress();
+        if (myAddress == null) {
+            return tryLocalHostOrLoopback(0).getAddress();
```

Comment:
Good do with a WARN or INFO log message at this point to the end user to indicate it is falling back to localhost as no suitable network address could be found.

Reply:
sure!

Reply:
Actually it already logs the error as part of `tryLocalHostOrLoopback()` so already taken care of :)

Reply:
I can change FINE log to WARNING in `tryLocalHostOrLoopback()`